src/main/java/seedu/address/ui/CustomerWindow.java

```diff
@@ -0,0 +1,117 @@
+package seedu.address.ui;
+
+import static seedu.address.ui.Map.WEBVIEW_WRAPPER;
+
+import java.util.logging.Logger;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.web.WebView;
+import javafx.stage.Stage;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.person.Customer;
+
+/**
+ * Controller for a help page
+ */
+public class CustomerWindow extends UiPart<Stage> {
+
+    public static final String DEFAULT_MESSAGE = "Customer Window";
+
+    private static final Logger logger = LogsCenter.getLogger(CustomerWindow.class);
+    private static final String FXML = "CustomerWindow.fxml";
+
+    @FXML
+    private Label name;
+    @FXML
+    private Label phone;
+    @FXML
+    private Label customerId;
+    @FXML
+    private Label email;
+    @FXML
+    private Label address;
+    @FXML
+    private WebView map;
+
+    /**
+     * Creates a new HelpWindow.
+     *
+     * @param root Stage to use as the root of the HelpWindow.
```

Comment:
Please rename this to Create a new Customer Window. And a few are yet to be changed from help to Customer.

Reply:
done!